### PR TITLE
Fixed autoupdate error after deleting a slide (Fixed #3066).

### LINKED ISSUE #3066: regression: importing motions issues tons of errors
OS offers an example CSV file to see how motions are added to a CSV import file. So let's test it:

- download the offered CSV example file
- use this file for an import

result: tons of error messages like these (this is just an excerpt):

2017-03-08 01:03:12,179 - ERROR - worker - Error processing message with consumer openslides.utils.autoupdate.send_data:
Traceback (most recent call last):
  File "D:\OpenSlides\2.1test\lib\site-packages\channels\worker.py", line 119, in run
    consumer(message, **kwargs)
  File "D:\OpenSlides\2.1test\lib\site-packages\openslides\utils\autoupdate.py", line 177, in send_data
    for element in projector.get_collection_elements_required_for_this(collection_element):
  File "D:\OpenSlides\2.1test\lib\site-packages\openslides\core\models.py", line 184, in get_collection_elements_required_for_this
    output.extend(element.get_collection_elements_required_for_this(collection_element, value))
  File "D:\OpenSlides\2.1test\lib\site-packages\openslides\agenda\projector.py", line 68, in get_collection_elements_required_for_this
    if collection_element == CollectionElement.from_values(Item.get_collection_string(), config_entry.get('id')):
  File "D:\OpenSlides\2.1test\lib\site-packages\openslides\utils\collection.py", line 30, in from_values
    full_data=full_data, information=information)
  File "D:\OpenSlides\2.1test\lib\site-packages\openslides\utils\collection.py", line 69, in __init__
    self.get_full_data()
  File "D:\OpenSlides\2.1test\lib\site-packages\openslides\utils\collection.py", line 205, in get_full_data
    self.full_data = self.get_access_permissions().get_full_data(self.get_instance())
  File "D:\OpenSlides\2.1test\lib\site-packages\openslides\utils\collection.py", line 177, in get_instance
    self.instance = query.get(pk=self.id)
  File "D:\OpenSlides\2.1test\lib\site-packages\django\db\models\query.py", line 385, in get
    self.model._meta.object_name
openslides.agenda.models.DoesNotExist: Item matching query does not exist.
2017-03-08 01:03:12,238 - ERROR - worker - Error processing message with consumer openslides.utils.autoupdate.send_data:
Traceback (most recent call last):
  File "D:\OpenSlides\2.1test\lib\site-packages\channels\worker.py", line 119, in run
    consumer(message, **kwargs)
  File "D:\OpenSlides\2.1test\lib\site-packages\openslides\utils\autoupdate.py", line 177, in send_data
    for element in projector.get_collection_elements_required_for_this(collection_element):
  File "D:\OpenSlides\2.1test\lib\site-packages\openslides\core\models.py", line 184, in get_collection_elements_required_for_this
    output.extend(element.get_collection_elements_required_for_this(collection_element, value))
  File "D:\OpenSlides\2.1test\lib\site-packages\openslides\agenda\projector.py", line 68, in get_collection_elements_required_for_this
    if collection_element == CollectionElement.from_values(Item.get_collection_string(), config_entry.get('id')):
  File "D:\OpenSlides\2.1test\lib\site-packages\openslides\utils\collection.py", line 30, in from_values
    full_data=full_data, information=information)
  File "D:\OpenSlides\2.1test\lib\site-packages\openslides\utils\collection.py", line 69, in __init__
    self.get_full_data()
  File "D:\OpenSlides\2.1test\lib\site-packages\openslides\utils\collection.py", line 205, in get_full_data
    self.full_data = self.get_access_permissions().get_full_data(self.get_instance())
  File "D:\OpenSlides\2.1test\lib\site-packages\openslides\utils\collection.py", line 177, in get_instance
    self.instance = query.get(pk=self.id)
  File "D:\OpenSlides\2.1test\lib\site-packages\django\db\models\query.py", line 385, in get
    self.model._meta.object_name
openslides.agenda.models.DoesNotExist: Item matching query does not exist.[2017/03/08 01:03:12] HTTP POST /rest/topics/topic/ 201 [0.13, 127.0.0.1:49543]

### PR DESCRIPTION
**Pull request by emanuelschuetze**

### CODE REVIEW

File: openslides/assignments/projector.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def get_collection_elements_required_for_this(self, collection_element, config_e
         output = super().get_collection_elements_required_for_this(collection_element, config_entry)
         # Full update if assignment changes because then we may have new
         # candidates and therefor need new users.
-        if collection_element == CollectionElement.from_values(Assignment.get_collection_string(), config_entry.get('id')):
+        if collection_element .collection_string == Assignment.get_collection_string() and collection_element.id == config_entry.get('id'):
```
Comment (normanjaeckel) [inline openslides/assignments/projector.py]: One whitespace should be removed before `.collection_string`.

File: openslides/motions/projector.py [hunk 2]
```diff
@@ -35,7 +35,7 @@ def get_collection_elements_required_for_this(self, collection_element, config_e
         #
         # Add some logic here if we support live changing of workflows later.
         #
-        if collection_element == CollectionElement.from_values(Motion.get_collection_string(), config_entry.get('id')):
+        if collection_element .collection_string == Motion.get_collection_string() and collection_element.id == config_entry.get('id'):
```
Comment (normanjaeckel) [inline openslides/motions/projector.py]: One whitespace should be removed before `.collection_string`.

File: openslides/assignments/projector.py (hunk 1 above)
Comment (emanuelschuetze) [inline openslides/assignments/projector.py]: done

File: openslides/motions/projector.py (hunk 2 above)
Comment (emanuelschuetze) [inline openslides/motions/projector.py]: done

**Comment by emanuelschuetze** [comment]
@normanjaeckel still open reference_projector

**Comment by normanjaeckel** [comment]
Travis failed. Rest is ok.

**Comment by emanuelschuetze** [comment]
Travis fixed.

**Pull request merged**